copyright

```diff
@@ -233,6 +233,68 @@ License: public-domain
  Taken from morgue-file.com, a collection of photographs that have been donated
  and placed in the public domain. (Exceptions noted below.)
 
+Files:
+ images/land/mercury1*
+Copyright: NASA/Johns Hopkins University Applied Physics Laboratory/Carnegie Institution of Washington
+License: public-domain
+Comment:
+ This image is an artist's impression of the surface
+ of Mercury, produced for the MESSENGER mission.
+ Because it was produced for a NASA mission, it is in
+ the public domain.
```

Comment:
```suggestion
 Taken from https://archive.is/dd4hA. This image is an
 artist's impression of the surface of Mercury, produced
 for the MESSENGER mission. Because it was produced for
 a NASA mission, it is in the public domain.
```